datadog/resource_datadog_monitor.go

```diff
@@ -260,6 +260,18 @@ func resourceDatadogMonitor() *schema.Resource {
 					Type:        schema.TypeBool,
```

Comment:
We should document that if there's a `custom_schedule` set, `require_full_window` is ignored.